test/integration/ruby_uninstall/inspec.yml

```diff
@@ -0,0 +1,9 @@
+name: rbenv
+title: rbenv profile
+maintainer: Sous Chefs
+copyright: Webb Agile Solutions Ltd.
```

Comment:
I just copied this file from the other suites. Is this still correct?

Reply:
If you wrote it as part of your company work, do that.

My copyright can't be on file file